scripts/operations/configuration/backup_cfs_config_comp.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+#
```

Comment:
Maybe I'm missing something but this seems like a lot of code to have to maintain just to reduce the number of command run from two to one.   Isn't this essentially just running "cray cfs configurations list" and "cray cfs components list"?

Reply:
Yeah. But we've had issues before where if we just include commands to redirect output to a file, we need to make sure that we aren't overwriting existing files, and are putting them in sensible locations. Similarly, there have been commands which failed in non-obvious ways, and most folks don't check return codes at the command line. Those are the two main reasons I favor automating even this kind of thing.

The other reason is that most of the functions defined in here are actually going to be in a common shell function library, but for 1.3 I didn't want to do that in the interest of limiting the size of the PR.

Reply:
I appreciate your attention to detail in your review, but my two biggest things I want your eyes for are:

1. Is it okay for me to leave the NCNs disabled in CFS until after they rebuild? For CSM it's fine, but do any of the other products need the updated NCN personalization to be run on the NCNs BEFORE they are rebuilt? 

2. Is this new procedure easier to follow?